app/src/main/java/com/example/memogram/SignUpPage.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.memogram
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.EditText
+import android.widget.Toast
+
+
+class SignUpPage : AppCompatActivity() {
+
+    lateinit var textFName : EditText
+    lateinit var textLName: EditText
+    lateinit var userName: EditText
+    lateinit var textEmail: EditText
+    lateinit var textPassword: EditText
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_sign_up_page)
+
+        textFName = findViewById(R.id.editTextFirstName)
+        textLName = findViewById(R.id.editTextLastName)
+        userName = findViewById(R.id.editTextUsername)
+        textEmail = findViewById(R.id.editTextEmail)
+        textPassword = findViewById(R.id.editTextPassword)
+    }
+
+    fun joinClick(view: View?){
+        val fname:String = textFName.text.toString()
+        val lname:String = textLName.text.toString()
+        val uname:String = userName.text.toString()
+        val email:String = textEmail.text.toString()
+        val password:String = textPassword.text.toString()
+
+
+        val bundle = Bundle()
+        bundle.putString("key1",fname)
```

Comment:
For this I think It is better if use the actual name for the keyString in the bundle. Like 

bundle.putString("fname", fname)

thanks! =)